src/lib/Surveys/surveyUtils.js

```diff
@@ -0,0 +1,121 @@
+import { showSurveyModal, surveyStore } from '$stores/surveyStore.js';
+
```

Comment:
Please write a couple basic unit tests for each of the functions in this file. That will help us ensure that there are no regressions going forward, and that we're all on the same page with regard to product behavior.